### PR TITLE
lollypop-portal: add python3

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
This [build failure](https://hydra.nixos.org/build/81614206) against https://github.com/NixOS/nixpkgs/pull/45950

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

cc @jtojnar

### CODE REVIEW

**Comment by xeji** [comment]
@GrahamcOfBorg build lollypop-portal

**Comment by worldofpeace** [comment]
Yeah, might as well for consistency across packages too.

File: pkgs/misc/lollypop-portal/default.nix [hunk 1]
```diff
@@ -19,42 +22,40 @@ stdenv.mkDerivation rec {
...
-      --prefix PYTHONPATH : "$program_PYTHONPATH"
       --prefix PATH : "${stdenv.lib.makeBinPath [ easytag kid3 ]}"
     )
   '';
 
   meta = with stdenv.lib; {
     description = "DBus Service for Lollypop";
-    homepage = https://gitlab.gnome.org/gnumdk/lollypop-portal;
-    license = licenses.gpl3Plus;
+    homepage    = https://gitlab.gnome.org/gnumdk/lollypop-portal;
```
Comment (jtojnar) [inline pkgs/misc/lollypop-portal/default.nix]: No need to align these.

Comment (worldofpeace) [reply pkgs/misc/lollypop-portal/default.nix]: Hah, not going to argue this again.

**Comment by jtojnar** [comment]
@GrahamcOfBorg build lollypop-portal

**Pull request merged**